From d2bae3e1637a0bf412c0223272bf402f66931f0b Mon Sep 17 00:00:00 2001
From: Ben Walton <bwalton@opencsw.org>
Date: Mon, 31 Oct 2011 03:51:55 +0100
Subject: [PATCH] Patch out a few usr/share references

The post-receive-email sample hook had usr/share references.  Correct
these for OpenCSW.

Signed-off-by: Ben Walton <bwalton@opencsw.org>
---
 contrib/hooks/post-receive-email |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/contrib/hooks/post-receive-email b/contrib/hooks/post-receive-email
index e459cc3..0849265 100755
--- a/contrib/hooks/post-receive-email
+++ b/contrib/hooks/post-receive-email
@@ -11,11 +11,11 @@
 # will have put this somewhere standard.  You should make this script
 # executable then link to it in the repository you would like to use it in.
 # For example, on debian the hook is stored in
-# /usr/share/git-core/contrib/hooks/post-receive-email:
+# /opt/csw/share/doc/git/contrib/hooks/post-receive-email:
 #
 #  chmod a+x post-receive-email
 #  cd /path/to/your/repository.git
-#  ln -sf /usr/share/git-core/contrib/hooks/post-receive-email hooks/post-receive
+#  ln -sf /opt/csw/share/doc/git/contrib/hooks/post-receive-email hooks/post-receive
 #
 # This hook script assumes it is enabled on the central repository of a
 # project, with all users pushing only to it and not between each other.  It
-- 
1.7.6.1

